src/Alert/Alert.stories.js

```diff
@@ -0,0 +1,194 @@
+    const message = text("Message", "The quick, brown fox jumps over a lazy dog.");
+    return {
+      info:
+        "Use when a user just performed some action and we need to tell them that action was successful. This button is usually used without an action button.\n" +
+        "Avoid using success banner if there is any follow-up action, for example, in cases where we need to confirm something to users by e-mail later. It's recommended to use informational alert instead. Visit Orbit.Kiwi for more detailed guidelines.",
+      chapters: [
+        {
+          sections: [
+            {
+              sectionFn: () => (
+                <ThemeProvider>
+                  <Alert type="success" title={renderTitle} icon>
+                    {message}
+                  </Alert>
+                </ThemeProvider>
+              ),
+              options,
+            },
+          ],
+        },
+      ],
+    };
+  })
+  .addWithChapters("Warning alert", () => {
+    const title = text("Title");
+    const renderTitle = title === "" ? undefined : title;
+    const message = text("Message", "The quick, brown fox jumps over a lazy dog.");
+    return {
+      info:
+        "Use in cases when you need to inform users about a potentially dangerous situation in their trip and it requires some action from them. However, if the issue requires immediate attention, use critical alert instead. Visit Orbit.Kiwi for more detailed guidelines.",
+      chapters: [
+        {
+          sections: [
+            {
+              sectionFn: () => (
+                <ThemeProvider>
+                  <Alert type="warning" title={renderTitle} icon>
+                    {message}
+                  </Alert>
+                </ThemeProvider>
+              ),
+              options,
+            },
+          ],
+        },
+      ],
+    };
+  })
+  .addWithChapters("Critical alert", () => {
+    const title = text("Title");
+    const renderTitle = title === "" ? undefined : title;
+    const message = text("Message", "The quick, brown fox jumps over a lazy dog.");
+    return {
+      info:
+        "Use when something is blocking users from continuing or when some issue needs to be resolved immediately. A critical alert should provide some form of solution for their problem. If something is important for users to solve as soon as possible, automatic open of a modal window is worthy of considering. Visit Orbit.Kiwi for more detailed guidelines.",
+      chapters: [
+        {
+          sections: [
+            {
+              sectionFn: () => (
+                <ThemeProvider>
+                  <Alert type="critical" title={renderTitle} icon>
+                    {message}
+                  </Alert>
+                </ThemeProvider>
+              ),
+              options,
+            },
+          ],
+        },
+      ],
+    };
+  })
+  .addWithChapters("Playground", () => {
+    const typeOptions = {
```

Comment:
Can we move this to component and export it ? I want to maintain this in one place